### PR TITLE
LPD-38774: Account restriction in object not filtering properly using Custom Facets

### PR DESCRIPTION
**Pull request by gustavolimav**
Ticket: https://liferay.atlassian.net/browse/LPD-38774

I need just a review of the object code. Tests are not included because this review is just to have hotfix approved

### CODE REVIEW

**Comment by gustavolimav** [comment]
ci:help

**Comment by gustavolimav** [comment]
ci:test:object

**Comment by gustavolimav** [comment]
ci:report:35762106

**Comment by marcelabc** [comment]
Just started reviewing :)

File: modules/apps/object/object-service/src/main/java/com/liferay/object/internal/search/spi/model/permission/ObjectEntrySearchPermissionFilterContributor.java [hunk 1]
```diff
@@ -50,16 +52,29 @@ public void contribute(
...
-			});
+				if (!accountEntry.isInactive()) {
+					accountEntryIds.add(accountEntry.getAccountEntryId());
+				}
+			}
+			catch (PortalException portalException) {
+				_log.error(portalException);
+			}
+		}
+
```
Comment (marcelabc) [inline modules/apps/object/object-service/src/main/java/com/liferay/object/internal/search/spi/model/permission/ObjectEntrySearchPermissionFilterContributor.java:77]: Can you change your code to this, please:
```
List<Long> accountEntryIds = new ArrayList<>();

for (AccountEntryUserRel accountEntryUserRel :
	_accountEntryUserRelLocalService.
		getAccountEntryUserRelsByAccountUserId(
			permissionChecker.getUserId())) {

	try {
		AccountEntry accountEntry =
			accountEntryUserRel.getAccountEntry();

		if (accountEntry.isInactive()) {
			continue;
		}

		accountEntryIds.add(accountEntry.getAccountEntryId());
	}
	catch (PortalException portalException) {
		_log.error(portalException);
	}
}
```

**Review by marcelabc — CHANGES_REQUESTED** [review]
Hi 
I finished my review. Let me know if you have questions :smile:

**Comment by marcelabc** [comment]
No related failures.

**Comment by marcelabc** [comment]
Hey @gustavolimav,
Please feel free to make the changes I asked for and send this PR to your team's GitHub. Is that okay?
Can you tag me when you send it to your team's GitHub? So I can double-check the changes.

**Comment by gustavolimav** [comment]
Sure! Thanks for the review @marcelabc! I will close this PR and tag you in the next one.

**Pull request closed without merging**